StarWars.Android/Views/CharactersFragment.cs

```diff
@@ -0,0 +1,48 @@
+using _Microsoft.Android.Resource.Designer;
+using Android.Views;
+using MvvmCross.Binding.BindingContext;
+using MvvmCross.DroidX.RecyclerView;
+using MvvmCross.Platforms.Android.Binding.BindingContext;
+using MvvmCross.Platforms.Android.Presenters.Attributes;
+using MvvmCross.Platforms.Android.Views.Fragments;
+using StarWars.Android.Adapters;
+using StarWars.Core.Const;
+using StarWars.Core.ViewModels;
+
+namespace StarWars.Android.Views;
+
+[MvxFragmentPresentation(ActivityHostViewModelType = typeof(MainViewModel), FragmentContentId = ResourceConstant.Id.content_frame)]
+public sealed class CharactersFragment : MvxFragment<CharactersViewModel>
+{
+    private MvxRecyclerView _charactersList;
+    public override View OnCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState)
+    {
+        base.OnCreateView(inflater, container, savedInstanceState);
+
+        var view = this.BindingInflate(ResourceConstant.Layout.characters_fragment, container, false);
+        
+        var spacingInPx = (int)(Resources.DisplayMetrics!.Density * Constants.SpacingDp + 0.5f);
+        
+        _charactersList = view?.FindViewById<MvxRecyclerView>(ResourceConstant.Id.charactersList);
+        _charactersList!.AddItemDecoration(new SpacingItemDecoration(spacingInPx));
+        _charactersList!.Adapter = new CharactersAdapter((IMvxAndroidBindingContext)BindingContext);
+
+        return view;
+    }
+
+    public override void OnViewCreated(View view, Bundle savedInstanceState)
+    {
+        base.OnViewCreated(view, savedInstanceState);
+        
+        var bindingSet = this.CreateBindingSet<CharactersFragment, CharactersViewModel>();
```

Comment:
I think you already need FragmentBase<T> to incapsulate this boilerplate logic with binding set creation